cmd/dep/testdata/harness_tests/init/godep/case2/testcase.json

```diff
@@ -0,0 +1,11 @@
+{
+  "commands": [
+    ["init", "-no-examples", "-skip-tools"]
+  ],
+  "error-expected": "",
+  "gopath-initial": {},
```

Comment:
Let's pre-populate the gopath with the dependencies. This will:

a. Give us control over the versions used, so that they don't collide with the same versions in case 1. Right now the final Gopkg.lock is exactly the same between the two cases, making it hard to tell that the `-skip-tools` flag worked.
b. Protect the test from failing when new releases are added to the test repositories. By omitting the repo from gopath, the test now depends on pulling the repo from the network.